src/Microsoft.Health.SqlServer/DefaultSqlConnectionStringProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Health.SqlServer.Configs;
+
+namespace Microsoft.Health.SqlServer
+{
+    /// <summary>
+    /// The default SQL connection string provider uses the connection string specified in configuration.
+    /// </summary>
+    public class DefaultSqlConnectionStringProvider : ISqlConnectionStringProvider
+    {
+        private readonly string _sqlConnectionString;
```

Comment:
I discovered an issue while testing with the SchemaManager tool:  By only storing the connection string here, the DefaultSqlConnectionStringProvider does not pick up changes made to the singleton SqlServerDataStoreConfiguration in the SchemaManager tool's command-handler code.  Having  DefaultSqlConnectionStringProvider store a reference to the SqlServerDataStoreConfiguration  singleton solves this issue.

Reply:
Fixed.